src/mnode-config.cpp

```diff
@@ -58,6 +58,13 @@ std::string get_string(json::iterator& it, std::string name)
         return it->at(name);
     return "";
 }
+
+unsigned int get_uint(json::iterator& it, std::string name)

```

Comment:
pass name parameter by const reference (const std::string &name). Here the copy of the "name" string is created.
Also, you are calling it->at(name) 3 times here.... check it->count separately,  if it is = 0, return.
Then just const auto &value = it->at(name);-> check it and return the value.
It would be good if for all new code we add some comments (you can use doxygen comments style).